### DISCUSSION TITLE
Maximum capacity dropped significantly after using AlDente for a week?

### DISCUSSION THREAD

**Discussion by baomdg** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1276] [Q&A]
I've been using AlDente for a week and set the limit at 75%.

When I downloaded AlDente, on the Stats section it shown that my Maximum Capacity of Battery Health is fluctuated around 97% and 98% (while on the Macbook Maximum Capacity is still 100).

However, after using AlDente for a week, I've just checked the Maximum Capacity on the app and saw that it dropped from 97%-98% to 93%!

I don't know if I'm using AlDente wrong or I don't understand about the Maximum Capacity feature of it. But it makes me very confused.

Please nicely enlighten me!

**Comment by MatthiasKerbl** [comment]
Hi @baomdg,

In general, the remaining capacity of MacBook batteries is just a good guess and not a static number. Therefore, it tends to fluctuate up and down a lot depending on various factors. Keeping your battery at a lower level without doing full cycles regularly can decrease the accuracy of the battery capacity readings. Therefore, we recommend doing full cycles or running the Calibration Mode regularly.

Best,
Matthias
